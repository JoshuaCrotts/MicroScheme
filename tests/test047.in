(define pi 3.141592653589793)
(define (circumference-proc r) (* 2 pi r))
(circumference-proc 5)

(define circumference-lambda (lambda (r) (* 2 pi r)))
(circumference-lambda 5)
