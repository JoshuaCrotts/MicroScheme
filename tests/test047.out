31.41592653589793
31.41592653589793
